packages/router-plugin/src/core/router-code-splitter-plugin.ts

```diff
@@ -203,9 +175,9 @@ export const unpluginRouterCodeSplitterFactory: UnpluginFactory<
         self.normalModuleFactory.hooks.beforeResolve.tap(
```

Comment:
You can remove all of this and just leave this (like with the Rspack config).
```ts
webpack(compiler) {
  ROOT = process.cwd()

  userConfig = getConfig(options, ROOT)
}
```

Reply:
done